src/OperationHandler.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Copyright MediaCT. All rights reserved.
+ * https://www.mediact.nl
+ */
+
+namespace Johmanx10\Transaction;
+
+use Johmanx10\Transaction\Visitor\OperationVisitorAwareInterface;
+use Johmanx10\Transaction\Visitor\OperationVisitorInterface;
+
+class OperationHandler implements
+    OperationHandlerInterface,
+    OperationVisitorAwareInterface
+{
+    /** @var OperationVisitorInterface[] */
+    private $visitors = [];
+
+    /**
+     * Handle the given operations.
+     *
+     * @param OperationInterface ...$operations
+     *
+     * @return void
+     */
+    public function handle(OperationInterface ...$operations): void
+    {
+        $transaction = new Transaction(...$operations);
```

Comment:
I could decouple this by introducing a factory for transactions and optionally injecting that in the constructor of the operation handler. Would you prefer this, @ashokadewit ?